src/main/groovy/com/bmuschko/gradle/docker/tasks/AbstractDockerRemoteApiTask.groovy

```diff
@@ -167,6 +172,17 @@ abstract class AbstractDockerRemoteApiTask extends DefaultTask {
         dockerClient
     }
 
+    private DockerCmdExecFactory getDockerCmdExecFactory() {
+        boolean useNettyExecFactory = Boolean.valueOf(getProject().findProperty('gdpNettyExecFactory').toString())
+            ?: Boolean.valueOf(System.getProperty('gdp.netty.exec.factory'))
+            ?: Boolean.valueOf(System.getenv('GDP_NETTY_EXEC_FACTORY'))
+
+        if (useNettyExecFactory) {
+            this.logger.debug("Using " + NettyDockerCmdExecFactory.class.simpleName + " as driver for " + DockerClient.class.simpleName)
```

Comment:
There's no need for the explicit usage of `this` here.

Reply:
Done.